src/ext.js

```diff
@@ -26,7 +26,23 @@ const toggleOptions = [
   },
 ];
 
+const fontSizeOptions = [
+  {
+    value: 'responsive',
+    translation: 'properties.responsive',
+  },
+  {
+    value: 'relative',
+    translation: 'properties.fluid',
+  },
+  {
+    value: 'fixed',
+    translation: 'properties.fixed',
+  },
+];
+
 export default function ext({ translator, shouldHide, senseNavigation }) {
+  const { isEnabled } = shouldHide;
```

Comment:
what is shouldHide? is that just that we don't want feature flag handling to be visible in open source projects?

Reply:
No shouldHide actually holds a couple of functions. From index.js:

```const shouldHide = { isEnabled, isFeatureBlacklisted, isUnsupportedFeature };```